src/js/worker/api/actions.js

```diff
@@ -138,6 +138,17 @@ const playMenu = {
         }
     },
 
+    throughRound: async (conditions: Conditions) => {
+	if (g.phase === PHASE.PLAYOFFS) {
+	    const playoffSeries = await idb.cache.playoffSeries.get(g.season);
+
+            const series = playoffSeries.series[playoffSeries.currentRound][0];
+            const numDaysThisSeries = 7 - series.home.won - series.away.won;
+
+            game.play(numDaysThisSeries, conditions);
```

Comment:
You're right, this won't quite work. This is actually why the feature doesn't exist, because it's hard to do right. The ideal solution would be to somehow communicate back to this function when a full round of playoffs actually ends (be it after 4, 5, 6, or 7 games). That could be done either with a return value from game.play or by an event emitter. Probably the return value would be better, but it'll be a little messy either way.

I'm going to close this PR because this implementation doesn't quite work, but I do want to add this feature eventually.